pkg/platform/runtime/artifact/buildPlan.go

```diff
@@ -0,0 +1,215 @@
+package artifact
+
+import (
+	"fmt"
+
+	"github.com/ActiveState/cli/internal/locale"
+	"github.com/ActiveState/cli/internal/logging"
+	"github.com/ActiveState/cli/internal/multilog"
+	model "github.com/ActiveState/cli/pkg/platform/api/graphql/model/buildplanner"
+	monomodel "github.com/ActiveState/cli/pkg/platform/model"
+	"github.com/go-openapi/strfmt"
+	"github.com/thoas/go-funk"
+)
+
+// ArtifactBuildPlan comprises useful information about an artifact that we extracted from a build plan
+type ArtifactBuildPlan struct {
```

Comment:
Isn't the buildplan irrelevant at this point? I understand the data that gets fed into this struct came from a buildplan, but the feeding seems entirely manual, and so in terms of this package and this code the buildplan seems irrelevant.

If that assumption is wrong; please clarify in comments how this relates to the buildplan beyond that this is where the data is sourced from.